src/main/SignUp/createUserApi.js

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios'
```

Comment:
Maybe this file should be called something more like `Users/UsersRequests/Requests`

Reply:
and test name

Reply:
done and I also reorganized them 👍